packages/circuit-ui/components/NotificationToast/NotificationToast.tsx

```diff
@@ -61,7 +61,7 @@ export type NotificationToastProps = HTMLAttributes<HTMLDivElement> &
     /**
      * A clear and concise description of the icon and the Toast's purpose. If the toast body is self-explanatory pass an empty string.
      */
-    iconLabel: string;
+    iconLabel?: string;
```

Comment:
You also need to add the default empty string on line 150: `iconLabel = '',`

Reply:
ugh thanks, completely forgot